### ISSUE TITLE
same dimensional feature output for all input videos of different lengths

### ISSUE THREAD

**Issue by xcyquan** [antoine77340/video_feature_extractor#18]
hi,  I am trying to run the feature extractor for my video pre-processing.

I want to generate the same dimensional feature output for all input videos of different lengths, is there an easy way to do this?

Thank you!

**Comment by Tereams** [comment]
yes. i encounter the same question, and i guess we have to find a min length of the input videos and adjust the code a little bit
